definition/s3.go

```diff
@@ -0,0 +1,65 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed With this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package definition
+
+import (
+	"errors"
+	"fmt"
+	"unicode/utf8"
+)
+
+// S3Grantee ...
+type S3Grantee struct {
+	ID          string `json:"id"`
+	Type        string `json:"type"`
+	Permissions string `json:"permissions"`
+}
+
+// S3 ...
+type S3 struct {
+	Name           string      `json:"name"`
+	ACL            string      `json:"acl"`
+	BucketLocation string      `json:"bucket_location"`
+	Grantees       []S3Grantee `json:"grantees"`
+}
+
+// Validate checks if a Network is valid
+func (s *S3) Validate() error {
+	aclTypes := []string{"full", "read", "read-acl", "write-acl"}
```

Comment:
Are Bucket_ACL-types == Grantee.Permissions ? In case it is we should rewrite the spec proposal defined [here](https://github.com/ernestio/ernest-site/pull/6/files#diff-9bbe41f23fc3a47f0343e44c673c0084R468)